backend/django_core/apps/anime/migrations/0011_animegenremodel_created_at_animegenremodel_is_locked_and_more.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2 on 2023-04-24 14:44
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("anime", "0010_animemodel_is_locked"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="animegenremodel",
+            name="created_at",
+            field=models.DateTimeField(
+                auto_now_add=True, default=django.utils.timezone.now
+            ),
```

Comment:
Instead this `default=django.utils.timezone.now`
I use
```py
field=models.DateTimeField(
        timezone.now, auto_now_add=True
),
```

Idk, is this a right way?